packages/medusa/src/api/auth/utils/generate-jwt-token.ts

```diff
@@ -1,8 +1,15 @@
+import { AuthIdentityDTO } from "@medusajs/types"
 import { generateJwtToken } from "@medusajs/utils"
 
 export function generateJwtTokenForAuthIdentity(
-  { authIdentity, actorType },
-  { secret, expiresIn }
+  {
+    authIdentity,
+    actorType,
+  }: { authIdentity: AuthIdentityDTO; actorType: string },
+  {
+    secret,
+    expiresIn,
+  }: { secret: string | undefined; expiresIn: string | undefined }
```

Comment:
```suggestion
  }: { secret?: string; expiresIn?: string }
```

Reply:
They are not optional, but they can be defined, so I think semantically it's better to use `undefined`